src/js/data.js

```diff
@@ -50,7 +51,9 @@ var data = {
       agent:          'uag'
     },
 
-    promo:            'code'
+    promo:            'code',
+
+    single_expire:    'sxp'
```

Comment:
Can we have an explanation of the meaning of this `single_expire` here? In a documentation, or maybe in the PR description? 🤔